exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,12 @@
-// Start by creating a variable `message`
-
+// Start by creating a variable `message` //
+let message = "Hello, my name is ";
```

Comment:
```suggestion
const message = "Hello, my name is ";
```
It's advisable to use **const** if we are not modifying the value